### ISSUE TITLE
Dialog not properly visible at 200% zoom

### ISSUE THREAD

**Issue by mharnetiaux** [microsoft/fluentui#17648]
### Environment Information

- All Browsers(200% zoom)

### Describe the issue:

This was called out in a HCL review.

1. Open a example Dialog in Fluentui.
2. Increase zoom level to 200% in Chrome.
3. Open dev tools and dock inspector to right side.
4. Drag screen below 500px.
5.  Notice dialog is not properly visible.

#### Actual behavior:

On small screens with 200% zoom, the dialog is not properly visible.

#### Expected behavior:

Dialog should be accessible on all screen sizes at 200% zoom.

**Comment by gouttierre** [comment]
@mharnetiaux - thanks for reaching out

@miroslavstastny - Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley

**Comment by miroslavstastny** [comment]
Unfortunately this repo currently contains multiple `Dialog` components. From the report I cannot see which Dialog is broken.
It would be a huge help if you could share package name and version which you use and screenshot of the broken state. Thanks for understanding.
